client/package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@pixi/filter-glow": "2",
+    "@pixi/filter-glow": "^2.7.0",
```

Comment:
I downgraded pixie originally to have a version that worked with our little older node project. Are we sure 2.7 is working fine? (No issues starting a game)

Reply:
I had no issues starting a game. I just installed it and this was the default. I didn't actively change the version.